src/test/java/gov/hvtesting/Hooks.java

```diff
@@ -27,6 +27,7 @@ public void BeforeSteps() {
 
     @Before("@TestData")
     public void PrepareTestData() {
+        dynamoDbApi = new DynamoDbApi();
```

Comment:
As said above, we'll need to refactor this and tidy up here. If we don't need direct access to read and write api's we will remove all the code. To revist.